### PR TITLE
Optimize the annotation of worker-size to avoid ambiguity

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/task-configuration-file-full.md [hunk 1]
```diff
@@ -27,21 +23,21 @@ DM 会根据任务类型进行相应检查。可以参考[关闭检查项](prech
...
+task-mode: all                  # 任务模式，可设为 "full" - "只进行全量数据迁移"、"incremental" - "Binlog 实时同步"、"all" - "全量 + Binlog 实时同步"
+shard-mode: "pessimistic"       # 任务协调模式，可选的模式有 ""、"pessimistic、"optimistic"。默认使用 "" 即无需协调。如果是分库分表合并任务，请设置为悲观协调模式 "pessimistic"。
+                                # 在 v2.0.6 版本后乐观模式逐渐成熟，深入了解乐观协调模式的原理和使用限制后，也可以设置为乐观协调模式 "optimistic"
 meta-schema: "dm_meta"          # 下游储存 `meta` 信息的数据库
 timezone: "Asia/Shanghai"       # 时区
 case-sensitive: false           # schema/table 是否大小写敏感
-online-ddl: true                # 目前支持 "gh-ost" 、"pt" 的自动处理
+online-ddl: true                # 支持上游 "gh-ost" 、"pt" 的自动处理
 online-ddl-scheme: "gh-ost"     # `online-ddl-scheme` 在未来将会被弃用，建议使用 `online-ddl` 代替 `online-ddl-scheme`
-ignore-checking-items: []       # 不关闭任何检查项。可选的检查项有 "all"、"dump_privilege"、"replication_privilege"、"version"、"binlog_enable"、"binlog_format"、"binlog_row_image"、"table_schema"、"schema_of_shard_tables"、"auto_increment_ID"
```
Comment (GMHDBJD) [inline zh/task-configuration-file-full.md:37]: 这个不暴露了吗？

**Review by GMHDBJD — COMMENTED** [review]
rest LGTM

File: zh/task-configuration-file-full.md (hunk 1 above)
Comment (sunzhaoyang) [reply zh/task-configuration-file-full.md:37]: 是的。期望搞成隐含参数，一般不让用，已经出现用户误用的情况
同时下个版本还要梳理下 检查项，即时要用一些检查项也不允许忽略

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: zh/task-configuration-file-full.md [hunk 2]
```diff
@@ -111,8 +107,8 @@ loaders:                             # load 处理单元的运行配置参数
 
 syncers:                             # sync 处理单元的运行配置参数
   global:                            # 配置名称
-    worker-count: 16                 # sync 并发迁移 binlog event 的线程数量，默认值为 16，当有多个实例同时向 TiDB 迁移数据时可根据负载情况适当调小该值
-    batch: 100                       # sync 迁移到下游数据库的一个事务批次 SQL 语句数，默认值为 100
+    worker-count: 16                 # sync 读取和应用已传输 binlog event 的并发线程数量，默认值为 16。调整此参数对上游压力无影响，对下游压力则显著影响。
```
Comment (Liuxiaozhen12) [inline zh/task-configuration-file-full.md:110]:
```suggestion
    worker-count: 16                 # sync 读取和应用已传输 binlog event 的并发线程数量，默认值为 16。调整此参数对上游压力无影响，对下游压力则影响显著。
```

File: zh/task-configuration-file-full.md [hunk 3]
```diff
@@ -111,8 +107,8 @@ loaders:                             # load 处理单元的运行配置参数
...
+    batch: 100                       # sync 迁移到下游数据库的一个事务批次 SQL 语句数，默认值为 100，一般不建议超过 500。
```
Comment (Liuxiaozhen12) [inline zh/task-configuration-file-full.md:111]:
```suggestion
    batch: 100                       # sync 迁移到下游数据库的一个事务批次 SQL 语句数，默认值为 100，建议一般不超过 500。
```

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**